apps/server/src/modules/files/uc/files.uc.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@nestjs/common';
+
+import { EntityId } from '@shared/domain';
+
+import { FilesService } from '../service/files.service';
+
+@Injectable()
+export class FilesUC {
+	constructor(private readonly service: FilesService) {}
+
+	async deleteUserData(userId: EntityId): Promise<void> {
+		await this.service.removeUserPermissionsToAnyFiles(userId);
+		await this.service.markFilesOwnedByUserForDeletion(userId);
```

Comment:
UC should check  authorizations / permissions

Reply:
This code is not used anywhere (it's not accessible by any _controller_ or anything), but good point, I'll add it as someone can forgot to add it once it will be accessible from some place.

Reply:
@SevenWaysDP sorry, after the discussion with our BA I've deleted the new _files_ module UC as we're still waiting for the complete requirements regarding the authorization layer for the deletion operations and because of that the UC with the authorization layer will be added in another ticket.